content/_partners/faq._en.md

```diff
@@ -141,26 +141,26 @@ development_accordion:
         id: c-3
         title: How do we get our application to production?
         content: >-
-            <a href="https://developers.login.gov/production/" class="external-link">See our production deployment page here</a>. We deploy changes to our production configuration weekly on Thursdays between 11am – 2pm EST. 
+            <a href="https://developers.login.gov/production/" class="external-link">Check Production deployment for more details.</a> We deploy changes to our production configuration weekly on Thursdays between 11am – 2pm EST. 
     -
         id: c-4
         title: What do you support-SAML vs. OAuth vs. OpenID?
```

Comment:
Unrelated to these other changes but I noticed it when looking, what if we use a colon or other question mark here? I'm not used to seeing a dash split a quesiton like this

```suggestion
        title: What do you support? SAML vs. OAuth vs. OpenID?
```

Reply:
@cdelivuk Are you okay with this content change?

Update heading from

> What do you support-SAML vs. OAuth vs. OpenID?

to

> What do you support? SAML vs. OAuth vs. OpenID?